src/devices/legrand.js

```diff
@@ -153,31 +153,6 @@ module.exports = [
             exposes.binary('led_in_dark', ea.ALL, 'ON', 'OFF').withDescription(`Enables the LED when the power socket is turned off,
             allowing to see it in the dark`)],
     },
-    {
-        // Some require coverInverted:
-        // - https://github.com/Koenkk/zigbee2mqtt/issues/15101#issuecomment-1356787490
-        // - https://github.com/Koenkk/zigbee2mqtt/issues/16090
-        fingerprint: [
```

Comment:
Why was this removed?

Reply:
That's correct, and it was not my intention to break anything.

Until shortly, this was required as there was no other alternative. Now that OTA is available, those devices work perfectly with the classic (non inverted) 067776.
I am preparing another PR with more extensive changes to the 067776, and wanted to avoid duplication.
The change has been reverted, hope that is OK.